src/en/news/blog/2022/three-large-scale-clusters/index.md

```diff
@@ -0,0 +1,250 @@
+testing scenarios. Testing Quincy at scale would allow us, for the first time, to identify bugs and
+bottlenecks that only occur on sizable clusters. We would also be able to evaluate a large cluster’s
+behavior in different environments, such as hardware, resource limitations, and workload patterns.
+
+In this blog post, we introduce three large-scale test clusters we worked with and outline the challenges
+and outcomes we encountered for each one. We also explain performance results that came as a result of our
+testing efforts. 
+
+## Gibba Upstream Cluster: Logical Large Scale
+
+The Gibba cluster is a large-scale test cluster that Ceph engineers periodically use to test upgrades and
+new features. Backed by Cephadm, the Gibba cluster holds a total of 975 OSDs (SSD) and 40 hosts. It also
+has 13 TiB of raw capacity.
+
+Initial challenges with the Gibba cluster included converting upstream teuthology nodes into a large-scale
+test cluster, dealing with limited CPU and memory resources, figuring out how to use tools like CBT (the Ceph
+Benchmarking Tool) at scale, and addressing bugs that led to redeployment.
+
+From our work on the Gibba cluster, we derived best tunings to continue large-scale testing, tested fresh
+deployments with Cephadm, fixed multiple bugs that were not caught in integration testing before releasing
+17.2.0, and made it possible for developers to perform large-scale tests on future releases.
+
+### Testing on Cephadm
+
+We encountered several challenges and outcomes on Gibba that were specific to Cephadm. The three main challenges
+we dealt with were 1) Rolling upgrades on a large scale from 16.2.7 to Quincy, 2) Managing upgrades with low
+resources, and 3) Running orch commands on many OSDs and hosts.
+
+To combat these challenges, we set up an Ansible playbook to install the Cephadm binary on each host to ensure
+that the hosts had the dependencies they needed, such as podman and lvm. We also tested a Cephadm purge at scale,
+which provided us with better knowledge of how to do this in future customer situations. Lastly, we identified and
+fixed six notable bugs in Cephadm, BlueStore, and manager modules that were not caught in upstream integration
+testing. See the [Tracker Index](#gibba%3A-testing-on-cephadm) for details.
+
+### Testing Telemetry
+
+After upgrading Gibba’s managers to Quincy, we were able to check the behavior of Telemetry’s new opt-in flow and
+the performance channel, both of which were new features added to Quincy. We also observed how the module behaved
+on a large scale.
+
+From our observations, we confirmed that the opt-in flow works as expected after an upgrade, fixed several bugs in
+the performance, device, and crash channels, introduced some fixes for the Dashboard behavior, and addressed an
+anonymization issue in the performance channel See the [Tracker Index](#gibba%3A-testing-telemetry) for details..
```

Comment:
Note to self: Accidentally put two periods here-- fix. Will leave for now since there will probably be additional edits.